perf/bloom_filter/bloom_filter.cc

```diff
@@ -0,0 +1,297 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+
+#include "bloom_filter.h"
+#include "msgpuck.h"
+#include "random.h"
+#include "trivia/util.h" /* For xmalloc. */
+
+#include "../utils/timer.h"
+
+/** Primary key id. */
+static constexpr std::uint32_t
+PK_IID = 0;
+/** Count of processed tuples.
+ *    2^14 -- OK time;
+ *    2^17 -- LONG time;
+ *    2^18 -- OK.
+ */
+/** Count of tuples for box insert warmup. */
+static constexpr std::uint32_t
+TUPLE_WARMUP_COUNT = 1 << 5;
+/**
+ * Size of random string to be inserted inside space.
+ * It has to be big enough so the dump occurs more often.
+ */
+static constexpr std::uint32_t
+STRING_FIELD_SIZE = 10000;
+/** Number of fields in inserted tuples. */
+static constexpr std::uint32_t
+TUPLE_FIELD_COUNT = 2;
+/** Tuple string for filling space. */
+static char tuple_string[STRING_FIELD_SIZE];
+/** File to save benchmark results. */
+static std::ofstream resultsFile;
+/** File name. */
+static std::string resultsFileName = "results.yml";
```

Comment:
This can throw, please use `const char []` or `string_view`.

Reply:
You can use a string view literal here and then pass it to `open` via `string_view.data`.